src/dailies.ts

```diff
@@ -143,7 +146,47 @@ export function postFreeFightDailySetup(): void {
 }
 
 function voterSetup(): void {
-  if (have($item`"I Voted!" sticker`) || !(get("voteAlways") || get("_voteToday"))) return;
+  if (have($item`"I Voted!" sticker`)) return;
+
+  if (!get("voteAlways") && get("_voteToday")) {
+    const valuableInitiatives: Map<string, number> = new Map([
+      [
+        "Meat Drop: +30",
+        0.3 *
+          ((baseMeat + 750) * embezzlerCount() + baseMeat * (estimatedTurns() - embezzlerCount())),
+      ],
+      [
+        "Item Drop: +15",
+        0.15 *
+          (4 * 100 * 0.3 * embezzlerCount() +
+            3 * 200 * 0.15 * (estimatedTurns() - embezzlerCount())),
+      ],
+      ["Adventures: +1", globalOptions.ascending ? 0 : get("valueOfAdventure")],
+    ]);
+    const availableInitiatives: Map<string, number> = new Map(
+      Object.keys(votingBoothInitiatives(myClass(), myPathId(), myDaycount())).map((s) => {
+        return [s, 0];
```

Comment:
Why iterate twice and not just set the value here?

Reply:
Not sure if the change was what you envisioned

Reply:
It was!